CHANGELOG.md

```diff
@@ -5,6 +5,16 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+## [2.27.0] - 2022-01-26
```

Comment:
Am I missing anything? Seems like there is no change in 2.27 version?

Reply:
Our new release process requires to create a new version ahead like the following 2.26.0 bump commit:
https://github.com/aws/amazon-chime-sdk-js/pull/1939/files

Reply:
I saw there are some entries in the `Fixed` and `Changed` session in the 2.26.0 bump, so I thought there should be some enties in the 2.27.0 bump. But I didn't go through the updated release script